lib/android/app/src/main/java/com/reactnativenavigation/anim/NavigationAnimator.java

```diff
@@ -68,9 +90,81 @@ public void onAnimationEnd(Animator animation) {
                 }
             }
         });
-        set.playTogether(translationY, alpha);
-        set.setDuration(DURATION);
-        set.setInterpolator(ACCELERATE_INTERPOLATOR);
         set.start();
     }
+
+    @NonNull
+    private AnimatorSet getDefaultPopAnimation(View view) {
+        AnimatorSet set = new AnimatorSet();
+
+        ObjectAnimator alpha = ObjectAnimator.ofFloat(view, View.ALPHA, 1, 0);
+        alpha.setInterpolator(ACCELERATE_INTERPOLATOR);
```

Comment:
Is there a reason we don't set shared properties once on the animator set?

Reply:
Not sure I understand the question